ee/tabby-ui/app/files/components/code-search-result.tsx

```diff
@@ -115,7 +115,11 @@ export const SourceCodeSearchResult = ({
         {ranges.map((range, index) => {
           const lines = result.lines.slice(range.start, range.end + 1)
           return (
-            <LazyLoad key={`${result.path}-${range.start}`} offset={300}>
+            <LazyLoad
+              height={lines.length * 20 + 9}
```

Comment:
Calculating the container height by the number of lines, this involves minimal changes.